lib/datasource/git-tags/index.js

```diff
@@ -1,41 +1,38 @@
-const { getRemoteInfo } = require('isomorphic-git');
+const simpleGit = require('simple-git/promise');
 
 const cacheNamespace = 'git-tags';
 const cacheMinutes = 10;
 
+// git will prompt for known hosts or passwords, unless we activate BatchMode
+process.env.GIT_SSH_COMMAND = 'ssh -o BatchMode=yes';
+
 async function getPkgReleases({ lookupName }) {
+  const git = simpleGit();
   try {
     const cachedResult = await renovateCache.get(cacheNamespace, lookupName);
     /* istanbul ignore next line */
     if (cachedResult) return cachedResult;
 
-    const info = await getRemoteInfo({
-      url: lookupName,
-    });
-    if (info && info.refs && info.refs.tags) {
-      const tags = Object.keys(info.refs.tags);
-      const result = tags.reduce(
-        (accum, gitRef) => {
-          if (!/\^/.test(gitRef)) {
-            // exclude things like '1.2.3^{}'
-            const version = gitRef.replace(/^v(?=[0-9])/, ''); // 'v1.2.3' => '1.2.3'
-            accum.releases.push({
-              version,
-              gitRef,
-            });
-          }
-          return accum;
-        },
-        {
-          sourceUrl: lookupName,
-          releases: [],
-        }
-      );
-      await renovateCache.set(cacheNamespace, lookupName, result, cacheMinutes);
-      return result;
-    }
+    // fetch remote tags
+    const lsRemote = await git.listRemote([
+      '--sort=-v:refname',
+      '--tags',
+      lookupName,
+    ]);
+    // get stable tags via regex. stable meaning only digits, dots and a leading 'v' will be allowed
+    const tags = String(lsRemote).match(/(v?[\d.]+)$/gm);
```

Comment:
Should we instead use `semver.isVersion()` ?

Reply:
And usually it's totally fine to have unstable tags retrieved because the filtering is done later.

Reply:
> Should we instead use `semver.isVersion()` ?

i could not found this in the semver package using `const semver = require('semver')`, am i missing something?

`git ls-remote` returns lines in form of `a749b99b00f6f26105048e6a650df046658f9d75	refs/tags/2.0.0-beta.26`, which i think cannot be transformed by semver directly (?), so we need to get rid of everthing before the tag. Unfortunately, javascript's `String.prototype.matchAll` is only available in nodejs 12+, so we have to use seperate calls to split results by line and then remove the overhead data. (Could also use [match-all](https://www.npmjs.com/package/match-all) but don't want to install a package for this single instance)

However i just realized that the current logic is wrong and will update it.